doc/weather-climate.rst

```diff
@@ -0,0 +1,159 @@
+.. _weather-climate:
+
+Weather and climate data
+========================
+
+.. ipython:: python
+   :suppress:
+
+    import xarray as xr
+
+``xarray`` can leverage metadata that follows the `Climate and Forecast (CF) conventions`_ if present. Examples include automatic labelling of plots with descriptive names and units if proper metadata is present (see :ref:`plotting`) and support for non-standard calendars used in climate science through the ``cftime`` module (see :ref:`CFTimeIndex`). There are also a number of geosciences-focused projects that build on xarray (see :ref:`related-projects`).
+
+.. _Climate and Forecast (CF) conventions: http://cfconventions.org
+
+.. _metpy_accessor:
+
+CF-compliant coordinate variables
+---------------------------------
+
+`MetPy`_ adds a	``metpy`` accessor that allows accessing coordinates with appropriate CF metadata using generic names ``x``, ``y``, ``vertical`` and ``time``. See `their documentation`_ for more information.
+
+.. _`MetPy`: https://unidata.github.io/MetPy/dev/index.html
+.. _`their documentation`:	https://unidata.github.io/MetPy/dev/tutorials/xarray_tutorial.html#coordinates
```

Comment:
ping @dopplershift for a quick peak at this section.

Reply:
That seems great to me. Thanks for adding. Would it also be appropriate to mention support for converting CF projection information to Cartopy projections?

Reply:
Yes! Can you send me some text or a "suggested change" then?

Reply:
Done.

Reply:
Thanks!